src/BloomExe/Book/Book.cs

```diff
@@ -371,6 +371,29 @@ public HtmlDom GetHtmlDomReadyToAddPages(HtmlDom inputDom)
 			return _storage.MakeDomRelocatable(inputDom, _log);
 		}
 
+		/// <summary>
+		/// Currently used by the image server
+		/// to get thumbnails that are used in the add page dialog. Since this dialog can show
+		/// an enlarged version of the page, we generate these at a higher resolution than usual.
+		/// Also, to make more realistic views of template pages we insert fake text wherever
+		/// there is an empty edit block.
+		/// </summary>
+		/// <param name="page"></param>
+		/// <returns></returns>
+		public Image GetThumbnailForPage(IPage page)
+		{
+			var pageDom = GetThumbnailXmlDocumentForPage(page);
+			var thumbnailOptions = new HtmlThumbNailer.ThumbnailOptions()
+			{
+				BackgroundColor = Palette.TextAgainstDarkBackground,
+				BorderStyle = HtmlThumbNailer.ThumbnailOptions.BorderStyles.Solid,
```

Comment:
That did help (allowed the pale blue border on the larger preview box), but needed to fix dimensions too.